docs/installation/production/installation.rst

```diff
@@ -118,3 +118,166 @@ Download and install any apps that you want to host using this installation of T
 .. todo::
 
     **Troubleshooting**: Here we try to provide some guidance on some of the most commonly encountered issues. If you are experiencing problems and can't find a solution here then please post a question on the `Tethys Platform Forum <https://groups.google.com/forum/#!forum/tethysplatform>`_.
+
+
+5. Setup SSL (https) on the Tethys Portal and Geoserver
```

Comment:
Change this title to:

5. Setup SSL (https) on the Tethys Portal and Geoserver (Recommended)

To indicate that it is not strictly necessary.